proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -267,6 +272,150 @@ message SelectOrUpdateRequest {
   repeated string column_specs = 3;
 }
 
+message Expression {
+  oneof type {
+    string column_name = 1;
+    int64 long_value = 2;
+    string raw = 3;
+  }
+}
+
+message Selectable {
+  string column_name = 1;
+  Expression expression = 2;
+}
+
+message Pair {
+  string output_column_name = 1;
+  string input_column_name = 2;
+}
+
+message MathContext {
+  enum RoundingMode {
+    UP = 0;
+    DOWN = 1;
+    CEILING = 2;
+    FLOOR = 3;
+    HALF_UP = 4;
+    HALF_DOWN = 5;
+    HALF_EVEN = 6;
+    UNNECESSARY = 7;
+  }
+  sint32 precision = 1;
+  RoundingMode rounding_mode = 2;
+}
+
+enum BadDataBehavior {
+  // Reset the state for the bucket to {@code null} when invalid data is encountered.
+  RESET = 0;
+
+  // Skip and do not process the invalid data without changing state.
+  SKIP = 1;
+
+  // Throw an exception and abort processing when bad data is encountered.
+  THROW = 2;
+
+  // Allow the bad data to poison the result. This is only valid for use with NaN.
+  POISON = 3;
+}
+
+message UpdateByRequest {
+
+  message Options {
+    optional bool use_redirection = 1;
+
+    optional int32 chunk_capacity = 2;
+
+    optional double maxStaticSparseMemoryOverhead = 3;
```

Comment:
snake case for message fields (maximumLoadFactor, targetLoadFactor too)

Reply:
Good catch.